### DISCUSSION TITLE
Add responses .png for different Route Params

### DISCUSSION THREAD

**Discussion by clabnet** [mockoon/mockoon#1437] [Help]
I need to return 3 different .png files using GET `/floorimages/:id` endpoint, where id can be a string (for example) "1", "2" OR "3". 

This work 
![image](https://github.com/mockoon/mockoon/assets/6067610/6f00d8c6-3ad5-4eb3-b684-25b3054ba33a)

Using rules I can specify only the RouteParams but I don't see how to return a different response for each id.

![image](https://github.com/mockoon/mockoon/assets/6067610/87ddd958-b57f-4c2e-ad13-d13286828451)

How to return different image for each :id param ?

Thank you in advance.

**Comment by 255kb** [comment]
With rules you would need as many responses as you have files. 
There is a simpler setup where you use the route param directly in the file name:

![image](https://github.com/mockoon/mockoon/assets/7489814/a367e30d-578a-46d4-b2af-d3500acfe5d4)

There is a recent tutorial on this topic: https://mockoon.com//tutorials/create-endpoint-serving-static-file/

**Reply by clabnet** [reply]
Very good tutorial, thank you.
